production/tanka/grafana-agent-operator/internal/logsinstance.libsonnet

```diff
@@ -0,0 +1,40 @@
+function(name='grafana-agent-logs', namespace='', config={}) {    
+    local this = self,
+    local gen = import 'agent-operator-gen/main.libsonnet',
+    local k = import 'ksonnet-util/kausal.libsonnet',
+    
+    local secret = k.core.v1.secret,
+    local li = gen.monitoring.v1alpha1.logsInstance,
+    local ga = gen.monitoring.v1alpha1.grafanaAgent,
+    local clients = li.spec.clients,
+
+    _config:: {
+        li_labels: {agent: "grafana-agent-logs"},
+        logs_rw_url: 'YOUR_HL_URL',
+        logs_rw_user: 'YOUR_HL_USER',
+        logs_rw_pass: 'YOUR_HL_PASS',
+        logs_secret_name: 'primary-credentials-logs',
+        external_labels: {cluster: 'cloud'}
+    } + config,
```

Comment:
In hindsight, these are all "optional" as well. Even though a config without remoteWrite seems useless, we don't want to impose that on the user.

They should be able to configure it without labels, without external labels, and without any remote write. Each of these can be `with{...}` functions. It would be good to allow multiple remoteWrite/client definitions as well.

Something [like this](https://github.com/grafana/agent/blob/main/production/tanka/grafana-agent/v1/lib/metrics.libsonnet#L87-L102).

This would all apply to the metrics instance as well.